v3/src/models/data/data-set.ts

```diff
@@ -409,13 +376,13 @@ export const DataSet = types.model("DataSet", {
 
         _childCases = []
         const lastCollectionGroup = newCollectionGroups.length
-                                      ? newCollectionGroups[newCollectionGroups.length - 1]
-                                      : undefined
+          ? newCollectionGroups[newCollectionGroups.length - 1]
+          : undefined
         if (lastCollectionGroup) {
           // if there are collections, then child cases are determined by the parents
           lastCollectionGroup?.groups.forEach(group => {
             _childCases.push(...group.childCaseIds.map((caseId, index) =>
-              ({__id__: caseId, [symParent]: group.pseudoCase.__id__, [symIndex]: index })))
+              ({ __id__: caseId, [symParent]: group.pseudoCase.__id__, [symIndex]: index })))
```

Comment:
Ops, these whitespace changes were done by VSC auto-formatter when I used it to format the code in the new block. I can revert the old one if it's a more common style in this repo.

Reply:
Preferences differ on whether there should be spaces inside braces so we allow both, but it should be consistent between the beginning and ending braces of the same construct, so I think this change is fine.